From 19221a0948ed51073cb5f31141760d91627bac0e Mon Sep 17 00:00:00 2001
From: Pascal Paillet <p.paillet@st.com>
Date: Wed, 3 May 2023 15:49:39 +0200
Subject: [PATCH 0093/1141] arm64: defconfig: Enable scmi regulator support

This patch adds in built-in the SCMI regulator driver
used by STM32MP SOCs.

Signed-off-by: Pascal Paillet <p.paillet@st.com>
Change-Id: Icf1a08d5e97ca11bb6af611267570aa88ca72607
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/304038
Reviewed-by: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Domain-Review: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
---
 arch/arm64/configs/fragment-02-defconfig-addons.config | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/fragment-02-defconfig-addons.config b/arch/arm64/configs/fragment-02-defconfig-addons.config
index 1574738d520b..f2ca36d2ee82 100644
--- a/arch/arm64/configs/fragment-02-defconfig-addons.config
+++ b/arch/arm64/configs/fragment-02-defconfig-addons.config
@@ -2,5 +2,6 @@
 CONFIG_ARCH_STM32=y
 CONFIG_ARM_SMC_WATCHDOG=y
 CONFIG_DYNAMIC_DEBUG=y
+CONFIG_REGULATOR_ARM_SCMI=y
 CONFIG_SERIAL_STM32=y
 CONFIG_SERIAL_STM32_CONSOLE=y
-- 
2.39.2

